runtime/syntax/c.vim

```diff
@@ -204,8 +204,8 @@ if exists("c_comment_strings")
   syn region cComment2String	contained start=+L\=\\\@<!"+ skip=+\\\\\|\\"+ end=+"+ end="$" contains=cSpecial
   syn region  cCommentL	start="//" skip="\\$" end="$" keepend contains=@cCommentGroup,cComment2String,cCharacter,cNumbersCom,cSpaceError,@Spell
   if exists("c_no_comment_fold")
-    " Use "extend" here to have preprocessor lines not terminate halfway a
-    " comment.
+    " Use "extend" here to have preprocessor lines not terminate halfway
+    " through a comment.
```

Comment:
We want to avoid modifying most `runtime/` files becaues it makes merges very easy, and we have no interest in maintaining them for the time being.